src/main/java/com/estsoft/guesshangeul/userrank/controller/ViewRankupRequController.java

```diff
@@ -0,0 +1,38 @@
+package com.estsoft.guesshangeul.userrank.controller;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.estsoft.guesshangeul.admin.entity.BoardManagerApply;
+import com.estsoft.guesshangeul.userrank.dto.RankupUpdateRequest;
+import com.estsoft.guesshangeul.userrank.dto.ViewRankupRequResponse;
+import com.estsoft.guesshangeul.userrank.repository.BoardManagerRepository;
+import com.estsoft.guesshangeul.userrank.service.ViewRankupRequService;
+
+@RestController
+public class ViewRankupRequController {//신청 받은 내용을 조회하기 위한 컨트롤러
+	private final ViewRankupRequService service;
+	@Autowired
+	private BoardManagerRepository repo;
+
+	public ViewRankupRequController(ViewRankupRequService service) {
+		this.service = service;
+	}
```

Comment:
의존성 주입 방식을 통일하면 좋을 것 같아요.
하나는 생성자 주입, 하나는 필드 주입이네요
생성자 주입으로 통일하신다 하면 다음과 같을 것 같습니다.
```suggestion
@RestController
@RequiredArgsConstructor
public class ViewRankupRequController {//신청 받은 내용을 조회하기 위한 컨트롤러
	private final ViewRankupRequService service;
	private final BoardManagerRepository repo;
```